packages/build/src/core/commands.js

```diff
@@ -217,6 +219,9 @@ const fireBuildCommand = async function({ buildCommand, configPath, buildDir, no
   }
 }
 
+// We use Bash on Unix and `cmd.exe` on Windows
+const SHELL = platform === 'win32' ? true : 'bash'
```

Comment:
Why not pass `true` for both of them?

Reply:
Because `true` resolves to `sh` (not `bash`) on Unix.

Reply:
Is that bad?

Reply:
The buildbot uses Bash to run `build.command`. Using `sh` breaks backward compatibility.

Reply:
I see.